Library/Face.cs

```diff
@@ -9,16 +9,20 @@ namespace BMeshLib
      * makes sense only 1. for clarity, because loops are a less intuitive
      * object and 2. to store face attributes.
      */
+    /// <summary>
+    /// Represents a face of a mesh in a <see cref="BMesh"/>.
```

Comment:
Again I think it is worth including the technical comment above as a remark. I feel it is important that users are well aware of the specifics of the data structure they are using.

Reply:
> Again I think it is worth including the technical comment above as a remark. I feel it is important that users are well aware of the specifics of the data structure they are using.

I disagree again on this one, but more so. The comment almost exclusively only explains and justifies the reason for the implementation. This information is not needed for an end user imo. It could also give the impression that the Face doesn't really need to be used.